pom.xml

```diff
@@ -701,7 +701,7 @@ Import-Package: \\
         </file>
       </activation>
       <properties>
-        <bnd.includeresource>${.}/../../lib/; filter:=*.jar; lib:=true</bnd.includeresource>
+        <bnd.includeresource>${.}/lib/; filter:=*.jar; lib:=true</bnd.includeresource>
```

Comment:
It will certainly fix the regression because it already used to be like this. 
See: https://github.com/openhab/openhab2-addons/pull/5211